.buildkite/react-native-cli-pipeline.yml

```diff
@@ -24,7 +24,7 @@ steps:
       docker-compose#v3.7.0:
         run: react-native-cli-tool-maze-runner
         use-aliases: true
-        command: ["-e", "features/built-app.feature"]
+        command: ["-e", "build-android-app.feature", "-e", "features/run-and-notify.feature"]
```

Comment:
I think it would be cleaner if I just moved the feature files into separate directories.  I'll do that as part of working on the iOS test.